### ISSUE TITLE
Why is the training accuracy okay, but the testing accuracy is very low?

### ISSUE THREAD

**Issue by Augety-Sun** [JeasunLok/ResNet-pytorch#1]

**Comment by JeasunLok** [comment]
The testing accuracy is low because the normalization has been used in training set but not used in testing set in the last version of my repository. I have revised my code in main.py to make sure the same transformation of the train set and val/test set.

**Comment by Augety-Sun** [comment]
Okay, thank you
